index.js

```diff
@@ -11,6 +11,38 @@ const pAccess = promisify(fs.access);
 // Path to included `xdg-open`.
 const localXdgOpenPath = path.join(__dirname, 'xdg-open');
 
+/**
+Get the mount point for fixed drives in WSL.
+
+@inner
+@returns {string} The mount point.
+*/
+const getWslDrivesMountPoint = (() => {
+	let mountPoint = null;
```

Comment:
```suggestion
	let mountPoint;
```

I prefer to use `undefined`.